### DISCUSSION TITLE
Links in tooltip go to wrong buffer

### DISCUSSION THREAD

**Discussion by AndreasMatthias** [emacs-lsp/lsp-mode#3995] [Q&A] [answered]
I'm working with `lua-language-server`, editing a file `test.lua` that looks like this:

```lua
function foo() end

---@see foo
function bar() end
```
Hovering over `bar()` a tooltip appears with a link to `foo()`. If I click on this link a new buffer call `test.lua#1#9` is opened which is wrong.

Note that function `foo()` is defined in buffer `test.lua` on line `1` and colum `9`. This would be the correct target of the link .

Is this an issue in `lsp-emacs` or `lua-language-server`?

**Comment by AndreasMatthias** [comment]
I've just tested it with VSCode. And it interprets the file link returned by lua-language-server correctly.

Are these kind of file links supported by lsp-emacs?

**Comment by yyoncho** [comment]
in general, sounds like a bug: we should handle that case here: https://github.com/yyoncho/lsp-mode/blob/master/lsp-mode.el#L4862

**Comment by yyoncho** [comment, marked as answer]
PR at https://github.com/emacs-lsp/lsp-mode/pull/3999/

**Reply by AndreasMatthias** [reply]
It's working for me. Thanks. :)
